### PR TITLE
Bug 1359878 fx lifestyle variations

### PR DESCRIPTION
**Pull request by jpetto**
## Description

Replacement for #4799. Includes a commit fixing issues found during first review, then another commit adding scene 2 templates.

~~Demo is throwing 400 errors for these variations. Do not merge until we can figure that out, but should be good for review.~~ @alexgibson is my hero.

## Bugzilla link

https://bugzilla.mozilla.org/show_bug.cgi?id=1359878

## Testing

Ran the following URLs through IE 9 - Edge and all looked pretty good.

https://bedrock-demo-jpetto.us-west.moz.works/en-US/firefox/new/?xv=privatenotoption
https://bedrock-demo-jpetto.us-west.moz.works/en-US/firefox/new/?xv=conformitynotdefault
https://bedrock-demo-jpetto.us-west.moz.works/en-US/firefox/new/?xv=browseuptoyou

## Checklist
- [ ] Requires l10n changes.
- [ ] Related functional & integration tests passing.

### CODE REVIEW

**Comment by alexgibson** [comment]
Medium viewports could do with a little love; some more bottom margin underneath the download button would be nice, and people in the background are also out of view?

medium-viewports

**Comment by alexgibson** [comment]
Is the empty white strip at the bottom of the download page intentional?

white-strip

File: media/css/firefox/new/fx-lifestyle/base.less [hunk 1]
```diff
@@ -0,0 +1,292 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../../sandstone/lib.less';
+
+@blue: #0a99d5;
+@grey: #EFEFEF;
```
Comment (jpetto) [inline media/css/firefox/new/fx-lifestyle/base.less]: Sigh, yeah, these variables are each only used once anyway...

File: media/css/firefox/new/fx-lifestyle/base.less [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+
+#outer-wrapper {
+    background: @grey;
+}
+
+#wrapper {
+    background: #fff;
+    width: auto;
+
+    // add icon to download buttons *not* in family nav
```
Comment (jpetto) [inline media/css/firefox/new/fx-lifestyle/base.less]: Remnants of initial commit - used to have styles for family nav, which was never displayed. 🤷‍♂️

File: media/css/firefox/new/fx-lifestyle/base.less [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+    .download-button .download-title:after {
+        background: url('/media/img/firefox/family/index-b/download-arrow.svg') center right no-repeat;
```
Comment (jpetto) [inline media/css/firefox/new/fx-lifestyle/base.less]: > the last set of temporary pages are still around

Good point. Will duplicate this image.

File: bedrock/firefox/templates/firefox/new/fx-lifestyle/base.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+{% block body_class %}{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+
+<section class="content-section" id="intro">
+  <div class="content">
+    <div class="copy">
+      <img src="{{ static('/media/img/firefox/new/firefox-logo.png') }}" />
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/new/fx-lifestyle/base.html]: Should this be just `img/firefox/new/firefox-logo.png`? Could this be the source of the error?

Comment (jpetto) [inline bedrock/firefox/templates/firefox/new/fx-lifestyle/base.html]: :rage4:

**Comment by jpetto** [comment]
I think we're ready for final review.

File: bedrock/firefox/templates/firefox/new/fx-lifestyle/base.html [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+      {% block head_content %}
+      {% endblock %}
+
+      <div id="download-button-wrapper-scene1">
+        {{ download_firefox( locale_in_transition=true, dom_id="download-firefox") }}
+      </div>
+
+      {# this button is used only for scene 2's #}
+      <div class="download-button-wrapper desktop" id="download-button-wrapper-desktop">
+        {{ download_firefox(force_direct=true, locale_in_transition=true, dom_id='primary-download-button') }}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/new/fx-lifestyle/base.html]: Hmm, this `locale_in_transition` shouldn't be needed for direct download links, right?

**Pull request merged**